vello/src/wgpu_engine.rs

```diff
@@ -827,7 +827,7 @@ impl WgpuEngine {
             label: Some(label),
             layout: Some(&compute_pipeline_layout),
             module: &shader_module,
-            entry_point: "main",
+            entry_point: Some("main"),
```

Comment:
I'd like to try:
```suggestion
            entry_point: None,
```
here

Reply:
Done.